assets/blocks/course-outline/module-block/radio-group-control.js

```diff
@@ -0,0 +1,41 @@
+import {
+	__experimentalRadio as Radio,
+	__experimentalRadioGroup as RadioGroup,
+	BaseControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * A component which controls the module status preview. Is contains a group of radio buttons and a description.
+ *
+ * @param {Object}   props                  Component props.
+ * @param {string}   props.previewStatus    The radio button that is selected.
+ * @param {Function} props.setPreviewStatus A callback which is called when a new option is selected.
+ */
+export const RadioGroupControl = ( { previewStatus, setPreviewStatus } ) => {
+	const id = 'inspector-radio-group-control';
```

Comment:
Should this be named based on what it is providing? `ModuleStatusControl`/`inspector-module-status-control` or something like that? We'll have a similar one for Lessons so it might be helpful to have the IDs and names diverge.

Reply:
I started to implemented this component as a reusable one but it proved to much work so I dropped the idea 😄  The names renamed though.

Reply:
Yeah, I suppose the actual component is namespaced enough that for now it is fine.